### ISSUE TITLE
Add line breaks to edd.css

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/Easy-Digital-Downloads#3073]
Since the `edd.css` file has a minified version, we should really add full line breaks after all rules in `edd.css` so that it's easier to read and modify. There's no reason to use single-line CSS here.

**Comment by chriscct7** [comment]
#3093 is ready for merge

**Comment by cklosowski** [comment]
This looks great @chriscct7. Love that you even left off the ```;``` on the last item in the styles :+1:, those add up to file size quick.

@sdavis2702 Can you peek over this with your CSS wizardry and just give your stamp of approval?

**Comment by mindctrl** [comment]
Since we ship a minified version, I'd like to see this version go "full on" standard. Spaces after the colons, ending with the semi-colons.
Instead of `display:block` I'd love `display: block;`.

Any reason to not do that here, other than it's annoying to update it now? :laughing:

**Comment by pippinsplugins** [comment]
I agree with @mindctrl

**Comment by pippinsplugins** [comment]
I'd also like to see selectors like this broken up into multiple lines:
```css
#edd_checkout_form_wrap textarea, #edd_checkout_form_wrap input[type="text"], #edd_checkout_form_wrap input[type="email"], #edd_checkout_form_wrap input[type="password"] {
```
to
```css
#edd_checkout_form_wrap textarea,
#edd_checkout_form_wrap input[type="text"],
#edd_checkout_form_wrap input[type="email"],
#edd_checkout_form_wrap input[type="password"] {
```

**Comment by pippinsplugins** [comment]
@chriscct7 Why did you remove the error and success classes?

**Comment by chriscct7** [comment]
What are you talking about? I didn't remove them. They're in the PR.

**Comment by pippinsplugins** [comment]
Ignore me. I was (not) seeing things.

**Comment by chriscct7** [comment]
This is done, with the spaces, and the breaking up the lines more, and the removing trailing semi-colons. I think we should strongly consider merging this into 2.3, as we could get merge conflict on future prs that touch the css. We can avoid that possibility completely if we merge it now, and it doesn't hurt to merge it now anyways. Plus it removes @pippinsplugins's awesome line of css: https://github.com/easydigitaldownloads/Easy-Digital-Downloads/blob/65163c94a5fac0128efe6e21cbb8e5dbab5eb2d3/templates/edd.css#L64

**Comment by sdavis2702** [comment]
@cklosowski Everything looks good to me. I don't really see the consistency in the "missing" semicolons, though. Some have it and some don't. Not a huge deal. I personally think they should always be there... especially in a file that is basically being handed to users.

**Comment by cklosowski** [comment]
Ok @pippinsplugins @sdavis2702 I'm going to merge this into ```release/2.3``` I think it's a good to go.

**Comment by chriscct7** [comment]
Merged into release/2.3. Closing.
